src/modules/imageresizer/tests/Properties/SettingsTests.cs

```diff
@@ -186,5 +186,93 @@ public void IDataErrorInfo_Item_returns_empty_when_not_JpegQualityLevel()
 
             Assert.Empty(result);
         }
-    }
+
+        [Fact]
+        public void Reload_createsFile_when_FileNotFound()
+        {
+            var settings = new Settings();
```

Comment:
nit: would be nice to have comment for `Arrange`, `Act` and `Assert` section. https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#arranging-your-tests

Reply:
I've added the same for all the tests I added. I haven't added it for the existing tests that were taken directly from the original repo since there are 72 tests that would need to be commented.